css/styles.css

```diff
@@ -0,0 +1,602 @@
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+.logo img {
+  display: none;
+  width: 150px;
+  height: 150px;
+}
+
+ul {
+  list-style: none;
+}
+
+.menu {
+  display: none;
+}
+
+.menu.active {
+  display: flex;
+  background-image: url(../images/bg-3.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  top: 0;
+  gap: 15px;
+  flex-direction: column;
+  align-items: center;
+  opacity: 0.8;
+  width: 100%;
+  height: 100vh;
+  text-align: center;
+  transition: 0.3s;
+  padding-top: 100px;
+  z-index: 1;
+}
+
+.contact {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.contact a {
+  color: #272a31;
+  cursor: pointer;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.nav-link a {
+  text-decoration: none;
+  color: #272a31;
+  font-weight: 700;
+  transition: 0.3s;
+  font-size: 1.1rem;
+}
+
+.nav-link a:hover {
+  color: #ed5342;
+}
+
+li:nth-child(6) {
+  border-width: 3px;
+  border-style: solid;
+  border-image: linear-gradient(to right, #ec5242, #272a31) 1;
+  padding: 7px;
+}
+
+section.hero {
+  display: flex;
+  flex-direction: column;
+  padding: 40px 30px;
+  height: 100vh;
+  margin-top: 60px;
+}
+
+.hero-text {
+  font-family: "Lato", sans-serif;
+  font-size: 1.3rem;
+  font-weight: 700;
+  color: #ed5342;
+}
+
+.hero-header {
+  font-family: inherit;
+  font-size: 2.5rem;
+  font-weight: bolder;
+  color: #ed5342;
+  text-transform: uppercase;
+  margin-bottom: 30px;
+}
+
+.text {
+  background-image: url(../images/text-bg-1.jpg);
+  background-repeat: repeat;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.description {
+  font-size: 1rem;
+  font-family: inherit;
+  padding: 15px;
+  margin-top: 10px;
+  line-height: 150%;
+  opacity: 0.7;
+}
+
+.header-h2 {
+  font-family: inherit;
+  text-transform: uppercase;
+  font-weight: bolder;
+}
+
+.main-program {
+  background-image: url(../images/program-bg.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 2rem;
+}
+
+.program-heading {
+  color: #fff;
+  font-size: 1.8rem;
+  background-color: #272932;
+  text-align: center;
+  padding-top: 30px;
+  position: relative;
+  margin-bottom: 3rem;
+}
+
+.program-heading::after {
+  content: "";
+  height: 3px;
+  width: 20%;
+  background: linear-gradient(to right, #d3d3d3, #f24432);
+  position: absolute;
+  margin: auto;
+  bottom: -5px;
+  left: 0;
+  right: 0;
+}
+
+.card-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 1rem;
+  justify-content: center;
+  align-items: center;
+}
+
+.card {
+  background-color: #b3b3b351;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.5rem;
+  width: 100%;
+  padding: 0.5rem;
+}
```

Comment:
- Hey, I really like your design. well done! I just want to point out something. Please, let's try to be attentive to details. This skill is very important in the dev world. Your card is 3, while the design is 5, can we add more cards to match the number? Please, I encourage you to do this.

![Screenshot from 2022-10-19 09-16-00](https://user-images.githubusercontent.com/59397159/196636292-26a33a63-4cd3-45d5-83fb-3bfdc50aeae1.png)

Reply:
- Also, the hover color of the card is `white`, let's update this also.

Reply:
Done